101-webapp-privateendpoint-vnet-injection/azuredeploy.json

```diff
@@ -0,0 +1,395 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "virtualNetwork_name": {
+      "type": "String",
+      "metadata": {
+        "description": "Name of the VNet"
+      }
+    },
+    "serverFarm_name": {
+      "type": "String",
+      "defaultValue": "ServerFarm1",
+      "metadata": {
+        "description": "Name of the Web Farm"
+      }
+    },
+    "site1_name": {
+      "type": "String",
+      "metadata": {
+        "description": "Web App 1 name must be unique DNS name worldwide"
+      }
+    },
+    "site2_name": {
+      "type": "String",

```

Comment:
add a defaultValue with uniqueString()

Reply:
I explicitly ask for the name of the vnet, added GEN in the parameter file

Reply:
Same as above, won't comment on the rest...

Reply:
done